src/Glpi/Controller/Session/ChangeProfileController.php

```diff
@@ -66,6 +66,13 @@ public function __invoke(Request $request): Response
         // Apply new profile
         Session::changeProfile($profile_id);
 
+        // If the profile change was made with an AJAX request, this mean this
+        // was some background script and we do not need to redirect it to
+        // another page.
+        if ($request->headers->contains('X-Requested-With', 'XMLHttpRequest')) {
+            return new Response(__("Profile changed successfully"));
```

Comment:
Either adapt the response to the expected content type, or an empty response as long as it is used only in tests.

```suggestion
            return new Response();
```